packages/database/src/models/PatientProgramRegistrationCondition.ts

```
@@ -55,6 +54,11 @@ export class PatientProgramRegistrationCondition extends Model {
       as: 'programRegistryCondition',
     });
 
+    this.belongsTo(models.ProgramRegistryCategory, {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)